pkg/deploy/server/server_configmap.go

```diff
@@ -308,15 +308,24 @@ func (s *Server) getCheConfigMapData() (cheEnv map[string]string, err error) {
 
 	addMap(cheEnv, s.deployContext.CheCluster.Spec.Server.CustomCheProperties)
 
-	// Update BitBucket endpoints
-	secrets, err := deploy.GetSecrets(s.deployContext, map[string]string{
+	err = setBitBucketEndpoints(s.deployContext, cheEnv)
```

Comment:
```suggestion
	err = setBitbucketEndpoints(s.deployContext, cheEnv)
```
let's be consistent with the way bitbucket names themselves.